addons/mail/models/mail_activity.py

```diff
@@ -465,6 +465,20 @@ def action_done(self):
         parameter, therefore setting context to feedback """
         return self.action_feedback()
 
+    def action_done_other_activities(self):
```

Comment:
Have to say that one is very cryptic, will leave it for the reviewer who has more context that me 😄